dynamic/discovery/discovery.go

```diff
@@ -131,6 +149,8 @@ func (rm *ResourceMap) refresh() {
 			// Make sure we don't choose a subresource like "pods/status".
 			if !strings.ContainsRune(apiResource.Name, '/') {
 				gve.kinds[apiResource.Kind] = apiResource
+			} else {
```

Comment:
Invert this if/else to get rid of the double negative, now that both branches have code.

Also update the comment about not choosing subresources to clarify, since we now do care about subresources. It should be something like:

```go
// Remember which resources are subresources, and map the kind to the main resource.
```